people/people_service.go

```diff
@@ -272,13 +270,15 @@ func (s service) Delete(uuid string) (bool, error) {
 		IncludeStats: true,
 	}
 
+	// Please note that this keeps the Identifiers as Identifiers are NOT a Thing
```

Comment:
I'm not sure this does keep the identifiers? I read this query as: Given a Thing (with uuid), that has no relationships to other Things, delete the node, and all its identifiers.